cvat-core/src/server-proxy.js

```diff
@@ -799,7 +799,7 @@
             }
 
             async function cancelLambdaRequest(requestId) {
-                const { backendAPI } = config;
+                const {backendAPI} = config;
```

Comment:
Please, setup eslint in your dev environent (install eslint extension for vscode and all npm packages).
Here is eslint error: ``eslintobject-curly-spacing``